shell/platform/android/android_shell_holder.cc

```diff
@@ -168,4 +261,37 @@ void AndroidShellHolder::NotifyLowMemoryWarning() {
   FML_DCHECK(shell_);
   shell_->NotifyLowMemoryWarning();
 }
+
+std::optional<RunConfiguration> AndroidShellHolder::BuildRunConfiguration(
```

Comment:
I moved this from the jni_impl to here since it's a lot of business logic and it would be nice to keep the jni as a simple pipe